packages/application-components/src/components/custom-views/custom-panel/custom-panel.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from 'react';
+import styled from '@emotion/styled';
+import { designTokens } from '@commercetools-uikit/design-system';
+import ModalPage from '../../modal-pages/internals/modal-page';
+
+type TCustomPanelProps = {
+  title: string;
+  size: 'small' | 'large';
+  onClose: () => void;
+  children: ReactNode;
+};
+
+const ContentWrapper = styled.div`
+  padding: ${designTokens.spacing40} 40px;
```

Comment:
Just wanted to double check if the horizontal padding should be out of our spacings scale.

Reply:
So this is coming from an `app-kit` [design token](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/application-components/src/theming.ts#L6) we will check if we want to keep or move to `ui-kit` in a follow-up task we have in our backlog.